README.rst

```diff
@@ -453,207 +527,276 @@ Navigating the Containers
 
 ``find(m,k)``
   Argument ``m`` is a SortedDict and argument ``k`` is a key.
-  This function returns a token that refers to the item whose key
+  This function returns the semitoken that refers to the item whose key
   is ``k``, or 
-  past-end marker if ``k`` is absent. Time: O(*c* log *n*)
+  past-end semitoken if ``k`` is absent. Time: O(*c* log *n*)
 
+``deref((m,s))``
```

Comment:
Where is this defined?  It seems to suggest that I pass in a tuple `(m,s)` to `deref`, but I only see definitions based on `Tokens` below.  

If it's that `(m,s)` is actually a `Token` type (is it?), then I would recommend writing it as `Token(m,s)` rather than using Julia's tuple syntax.